armbian/base/config/redis/factorysettings.txt

```diff
@@ -14,6 +14,8 @@ SET base:update:allow-unsigned 0
 SET base:updating 0
 SET base:setup 0
 
+SET hsm:firmware:version xxx
```

Comment:
imho I would not store it in redis, this produces follow up effort in maintaining it when upgrading, being careful about that it is updated atomically with the firmware, etc. This is only good if we know concretely what to use it for.

Furthermore, the middleware knows the version of the HSM when it connects to it.

Reply:
Removed the Redis key/value pair.